app/src/main/java/com/jerboa/ui/components/settings/SettingsActivity.kt

```diff
@@ -41,16 +41,6 @@ fun SettingsActivity(
         },
         content = { padding ->
             Column(modifier = Modifier.padding(padding)) {
-                SettingsMenuLink(
```

Comment:
This change seems unrelated

Reply:
i moved this back to its original location